### ISSUE TITLE
error bundling with rollup 0.36.1

### ISSUE THREAD

**Issue by davelandry** [d3/d3-hierarchy#59]
When creating a custom bundle using [rollup](https://github.com/rollup/rollup) 0.36.1 and the treemap function exported by d3-hierarchy, I get the following error in the browser console:

```js
Uncaught TypeError: Cannot create property 'y0' on number '1.618033988749895'
```

Which points to the following line:
https://github.com/d3/d3-hierarchy/blob/master/src/treemap/index.js#L20

It only happens when I initialize the treemap function. If I comment out the line `treemap()` it bundles correctly.
```js
import {treemap} from "d3-hierarchy";

export default function() {
  treemap();
}
```

Not sure if this is a d3-hierarchy bug or a [rollup](https://github.com/rollup/rollup) bug, so I apologize if this issue is misplaced.

**Comment by mbostock** [comment]
Sorry, but this isn’t enough information for me to investigate. What is the entry point to the bundle? The code you’ve included doesn’t look valid, because it’s calling the [treemap constructor](https://github.com/d3/d3-hierarchy/blob/master/README.md#treemap) and then discarding the result. Probably the error lies in other code, most likely the *root* node you’re passing to the *treemap* instance.

When reporting an issue, please include a link to a live example, preferably on [bl.ocks.org](http://bl.ocks.org) or [RunKit](https://runkit.com/npm/d3), or as a pull request in the [appropriate repository](https://github.com/d3) with new unit tests, that demonstrates that the described behavior is not the [expected behavior](https://github.com/d3/d3/blob/master/API.md). Use only the minimum amount of code necessary to reproduce the unexpected behavior.

A good bug report should **isolate specific methods** that exhibit unexpected behavior and precisely define how expectations were violated. What did you expect the method or methods to do, and how did the observed behavior differ? The more precisely you isolate the issue, the faster I can investigate.

Non-actionable bugs may be temporarily closed until you provide additional information. If you can’t isolate the bug any further, please let me know and I will try to help. However, I beg your patience: I must balance your request against many other responsibilities—fixing other bugs, answering other questions, new features, new documentation, *etc.* Please also consider asking for help on [Stack Overflow](https://stackoverflow.com/tags/d3.js).

If you have a question about D3’s behavior and want to discuss it with other users, also consider the [d3-js Google Group](https://groups.google.com/forum/#!forum/d3-js) or [joining](https://d3-slackin.herokuapp.com/) the [d3-js Slack](https://d3js.slack.com/).

Thank you! 🤗

**Comment by davelandry** [comment]
Fair enough. I can't imagine how much you must be bombarded every day with crap posts. Sorry to add to the clutter.

I've created a self-contained repo that shows the error: https://github.com/davelandry/rollup-d3

The only thing I can think of that I forgot to mention in the initial post is that I'm using [rollup-plugin-node-resolve](https://github.com/rollup/rollup-plugin-node-resolve). At the risk of cross-pollonation, I've also posted this issue in the rollup repository (as I'm starting to think that may be the culprit): https://github.com/rollup/rollup/issues/1009
